docs/reference_guide.md

```diff
@@ -1284,7 +1286,45 @@ listed first, the members are specific to the tracepoint.
 Examples in situ:
 [search /tools](https://github.com/iovisor/bpftrace/search?q=tracepoint%3A+path%3Atools&type=Code)
 
-## 7. `usdt`: Static Tracing, User-Level
+## 7. `rawtracepoint`: Static Tracing, Kernel-Level
+
+The hook point triggered by `tracepoint` and `rawtracepoint` is the same.
```

Comment:
Maybe also add:
```suggestion
The hook point triggered by `tracepoint` and `rawtracepoint` is the same. The difference is that raw tracepoints are faster since no argument processing is done (see below).
```